### PR TITLE
[rhosp 13] add change list

### PR DESCRIPTION
**Pull request by yong-huang**

### CODE REVIEW

File: cinder/Changelist.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+| Tags | Changes | Build date | Build type |
+| ---- | ------- | ---------- | ---------- |
+| 13.0-4-12 | No changes | 2019-1-25 | Auto |
+| 13.0-4-11 | Upgrade storops version from 0.5.11 to 1.0.1 | 2019-1-17 | Manual |
```
Comment (amagicsun) [inline cinder/Changelist.md]: How about the older versions?

Comment (yong-huang) [reply cinder/Changelist.md]: I can't get the tag info from Red Hat portal now, maybe there are some issues in Red Hat system, will update the older version info once I get the details.

Comment (amagicsun) [reply cinder/Changelist.md]: OK, please add a TODO here so that we won't forget.

Comment (yong-huang) [reply cinder/Changelist.md]: Done, all tags added.

**Pull request merged**